### ISSUE TITLE
not issue, just a question

### ISSUE THREAD

**Issue by NoUserNameForYou** [iAmMe27/Tahrovin#14]
I'm on oculus quest 2 and I tried both airlink/cable and virtual desktop and the performance is horrendous, like 15 fps on a 970gtx (as expected). Some guys told me I should try opencomposite but I'm wondering if it would break any mods that rely on SteamVR framework?

**Comment by NoUserNameForYou** [comment]
No, thanks. Discord and other leftist-infested mediocre IRC left-overs are not "scripta manent". If somebody finds an issue with your compilation they'll probably open an issue HERE so more people can benefit from the knowledge without registering on some 3rd party site.

**Comment by iAmMe27** [comment]
This was not an issue, it was a question, therefore it is not appropriate to be opened here. 

If you do not wish to join the Discord server for support, that is up to you but please do not bring political talk/beliefs here. This channel of communications should be low-noise to ensure that issues are attended to.
